project/src/js/lichess/friends.js

```diff
@@ -1,8 +1,40 @@
-import { caseInsensitiveSort } from '../utils';
-
 var onlineFriends = [];
 
+var makeUser = function(name, isPlaying) {
+    return {
+      'name' : name,
+      'isPlaying' : isPlaying,
+    }
+};
+
+/** Compares usernames for equality, ignoring prefixed titles (such as GM) */
+var isSameUser = function(userId, name) {
```

Comment:
Are you sure? Example 'following_onlines' payload from the server (featuring with player titles as the prefix to their usernames such as IM ChessExplained):

{"t":"following_onlines","d":["CM Kingscrusher-YouTube","cshuenss","NoJoke","MagicMann","thibault","KM2200","Clarkey","IM ChessExplained"],"playing":["kingscrusher-youtube","km2200"],"patrons":["kingscrusher-youtube","nojoke","thibault","clarkey","chessexplained"]}

(Note: I copied the above function from lichess' handler for following_onlines)